dts/arm64/broadcom/bcm2712.dtsi

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2024 Myeonghyeon Park <myeonghyeon@tsnlab.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm64/armv8-a.dtsi>
+#include <zephyr/dt-bindings/interrupt-controller/arm-gic.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-a76";
+			reg = <0>;
+		};
+	};
+
+	interrupt-parent = <&gic>;
+
+	timer {
+		compatible = "arm,armv8-timer";
+		interrupts = <GIC_PPI 13 IRQ_TYPE_LEVEL
+					  IRQ_DEFAULT_PRIORITY>,
+					 <GIC_PPI 14 IRQ_TYPE_LEVEL
+					  IRQ_DEFAULT_PRIORITY>,
+					 <GIC_PPI 11 IRQ_TYPE_LEVEL
+					  IRQ_DEFAULT_PRIORITY>,
+					 <GIC_PPI 10 IRQ_TYPE_LEVEL
+					  IRQ_DEFAULT_PRIORITY>;
+	};
+
+	soc {
+		#address-cells = <2>;
+		#size-cells = <1>;
+
+		sram0: memory@200000 {
+			device_type = "memory";
+			compatible = "mmio-sram";
+			reg = <0x0 0x200000 0x80000>;
+		};
+
+		gic: interrupt-controller@107fff9000 {
+			compatible = "arm,gic-v2", "arm,gic";
+			reg = <0x10 0x7fff9000 0x1000>,
+				  <0x10 0x7fffa000 0x2000>;
+			interrupt-controller;
+			#interrupt-cells = <4>;
+			status = "okay";
+		};
+
+		gpio2@107d517c00 {
+			compatible = "simple-bus";
+			reg = <0x10 0x7d517c00 0x40>;
+
+			#address-cells = <1>;
+			#size-cells = <0>;
+			gio_aon: gpio@0 {
```

Comment:
Why `gio_aon` is placed inside a `gpio2` node? https://github.com/raspberrypi/linux/blob/477f26fb73b8c73dea6503f62992d53ce0f3b00c/arch/arm/boot/dts/broadcom/bcm2712.dtsi#L626-L637

Reply:
I wrote it like this to separate these two banks: https://github.com/raspberrypi/linux/blob/477f26fb73b8c73dea6503f62992d53ce0f3b00c/arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts#L642-L669 
I thought it would be more convenient if I have to implement the second one. I'll fix it if this is not desirable.

Reply:
OK, I got it.